freeipa/src/main/resources/freeipa-salt/salt/nginx/conf/ssl.conf

```diff
@@ -32,6 +32,9 @@ server {
         ssl_verify_client on;
     {%- endif %}
     ssl_protocols TLSv1.2 TLSv1.3;
+{% if salt['pillar.get']('cluster:gov_cloud') == True %}
```

Comment:
you should have the default value as false, just to ensure if the pillar is not there, same for the others